ApollosProject/apollos-apps
# fix: add CurrentVideo context and pass data to onEnd

### **Purpose**
The purpose of this PR is to prevent the app from crashing by ensuring handleOnEnd receives required data like id.

@vinnyjth Please let me know if there's a better way to do this... It seems like there should be, but I'm blanking.

### **Testing**
1. Check out fix-crash-onEnd
2. Start up any multitenant client (I used newspring)
3. Add console logs for id, elaspedTime, and title inside of [handleOnEnd](https://github.com/ApollosProject/apollos-apps/blob/68f9a5e6583a8bc7855f0d67ccab0c28f26b397f/packages/apollos-ui-connected/src/ApollosPlayerConnected/index.js#L66)
4. With the app running open any video, scrub to a few seconds before the end and let it complete
5. You should see:
- the items get logged to your terminal
- the app no longer crashes

Below is a quick screencap showing that handleOnEnd receives the info it's looking for by logging the id, elapsedTime, and title to my terminal on the left.

https://user-images.githubusercontent.com/19194337/187772211-14066c51-e2d1-43f1-938d-4e3e315f3b17.mov
